Core/EntityManager/src/main/java/eu/neclab/ngsildbroker/entityhandler/controller/EntityController.java

```diff
@@ -147,6 +159,8 @@ public EntityController() {
 			UpdateResult update = entityService.partialUpdateEntity(HttpUtils.getHeaders(request), entityId,
 					expandedAttrib, expandedPayload);
 			logger.trace("partial-update entity :: completed");
+			String tenantid = HttpUtils.getInternalTenant(HttpUtils.getHeaders(request));
```

Comment:
[Major] removed this line from here and pass "HttpUtils.getHeaders(request)" parameter in patchtoEndpoint method instead of "tenantid" and after that tenantid get inside EntityService class.